templates/default/choose/chooselocation.html

```diff
@@ -209,7 +210,15 @@
 			street.value ? street.options[street.selectedIndex].text : '',
 			house.value, flat.value);
 	}
-
+        
+        var geocoder = new google.maps.Geocoder();
+	geocoder.geocode( { 'address': loc.value}, function(results, status) {
+            if (status == google.maps.GeocoderStatus.OK) {
```

Comment:
A to nie zadziała tak, że za każdym razem będzie wstawiać współrzędne z google maps nawet jeśli już są ustawione?